### PR TITLE
OPENEUROPA-2943: Upgrade to PHP 7.3.

### PR DESCRIPTION
**Pull request by sergepavle**
### Description

[Insert description here]

### Change log

- Added:
- Changed:
- Deprecated:
- Removed:
- Fixed:
- Security:

### Commands

```sh
[Insert commands here]

```

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -68,6 +69,7 @@
         }
     },
     "config": {
-        "sort-packages": true
+        "sort-packages": true,
+        "discard-changes": true
```
Comment (ademarco) [inline composer.json]: Why is this necessary?

Comment (sergepavle) [reply composer.json]: Exist issue on drone side (on composer install).
For not completely reasons we have on lower dependencies patched already drupal/coder module and we have error like that:

> RuntimeException]                                                                  
> Source directory /test/oe_translation/vendor/drupal/coder has uncommitted changes.  

By these settings, we enforce composer discard changes and discard changes (patched files)
